package.json

```diff
@@ -16,9 +16,9 @@
     "api-docs:lint": "spectral lint -F hint -v docs/api_v0.yml && lint-openapi -e docs/api_v0.yml",
     "api-docs:serve": "redoc-cli serve docs/api_v0.yml --options.pathInMiddlePanel --options.jsonSampleExpandLevel=all --options.menuToggle -t docs/api_template.hbs --watch",
     "storybook-prerequisites": "sass -q app/assets/stylesheets/themes:app/javascript/storybook-static/themes && bin/generate-css-utility-classes-docs.js",
-    "prebuild-storybook": "npm run storybook-prerequisites",
+    "prebuild-storybook": "yarn run storybook-prerequisites",
```

Comment:
nitpick (non-blocking): `yarn run` is not necessary. You can use just `yarn`, e.g. `yarn storybook-prerequisites` instead of `yarn run storybook-prerequisites`